aspnetcore/razor-components/get-started.md

```diff
@@ -5,58 +5,167 @@ description: Learn how to get started with the Razor Components framework.
 monikerRange: '>= aspnetcore-3.0'
 ms.author: riande
 ms.custom: mvc
-ms.date: 01/29/2019
+ms.date: 02/03/2019
 uid: razor-components/get-started
 ---
 # Get started with Razor Components
 
+By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.com/guardrex)
+
 [!INCLUDE[](~/includes/razor-components-preview-notice.md)]
 
-## Setup
+# [Visual Studio](#tab/visual-studio)
+
+Prerequisites:
+
+[!INCLUDE[](~/includes/net-core-prereqs-vs-3.0.md)]
+
+To create your first Razor Components project in Visual Studio:
+
+1. Select **File** > **New Project** > **Web** > **ASP.NET Core Web Application**.
+1. Make sure **.NET Core** and **ASP.NET Core 3.0** are selected at the top.
+1. Choose the **Razor Components** template and select **OK**.
+
+   ![New app dialog](https://msdnshared.blob.core.windows.net/media/2019/01/razor-components-template.png)
 
-Install the following:
+1. Press **Ctrl-F5** to run the app *without the debugger*. Running with the debugger (**F5**) isn't supported at this time.
+
+Congrats! You just ran your first Razor Components app!
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+Prerequisites:
 
-1. [.NET Core 2.1 SDK](https://go.microsoft.com/fwlink/?linkid=873092) (2.1.500 or later).
-1. [Visual Studio 2017](https://go.microsoft.com/fwlink/?linkid=873093) (15.9 or later) with the *ASP.NET and web development* workload selected.
-1. The latest [Blazor Language Services extension](https://go.microsoft.com/fwlink/?linkid=870389) from the Visual Studio Marketplace.
-1. The Blazor templates on the command-line:
+[!INCLUDE[](~/includes/net-core-prereqs-vsc-3.0.md)]
+
+To create your first Razor Components project in Visual Studio Code:
+
+1. Execute the following command from a command shell:
 
    ```console
-   dotnet new -i Microsoft.AspNetCore.Blazor.Templates
+   dotnet new razorcomponents -o WebApplication1
    ```
 
-To create your first project from Visual Studio:
+1. Open the *WebApplication1* folder in Visual Studio Code.
 
-1. Select **File** > **New Project** > **Web** > **ASP.NET Core Web Application**.
-1. Make sure **.NET Core** and **ASP.NET Core 2.1** are selected at the top.
-1. Choose the Blazor template and select **OK**.
+1. Add a *.vscode* folder.
```

Comment:
Doesn't VSCode add this stuff for you? You shouldn't need to add it manually . . .

Reply:
No ... not for a solution folder. These files need to be manually added unless the dev were to open either the *.App* or *.Server* folder directly in the IDE. I guess they detect that's in an app from the project file (guessing).